src/App.js

```diff
@@ -0,0 +1,3 @@
+const head = React.createElement("h1",{},"Hello world....");
```

Comment:
It could be better if you write your intuition in code comments , just as a learning booster

Reply:
Ok Tarun, I will follow it from next time. Thanks